packages/react-examples/src/react-charting/VerticalStackedBarChart/VerticalStackedBarChart.Callout.Example.tsx

```diff
@@ -23,6 +23,7 @@ interface IVerticalStackedBarState {
   selectedCallout: 'singleCallout' | 'MultiCallout' | 'MultiCustomCallout' | 'singleCustomCallout';
   showLine: boolean;
   barGapMax: number;
+  barWidth: number;
```

Comment:
I dont see any change in *BarChartPage.tsx files

Reply:
![image](https://user-images.githubusercontent.com/110246001/220534529-78102e23-d99c-4ff3-9d3f-bd0d257f8d78.png)
![image](https://user-images.githubusercontent.com/110246001/220534631-3ca8e3a1-7525-47f4-9cc5-76fd991717a5.png)
![image](https://user-images.githubusercontent.com/110246001/220534707-ff4039c0-cafe-4585-865a-46d702827542.png)